### PR TITLE
Rotate map elements

### PR DESCRIPTION
**Pull request by Broxzier**
This PR adds the option to rotate some map elements (Exceptions being surface, large scenery and banners) that was proposed in #3106, and fixes a few issues that the tile inspector had, mostly invalidation-wise.

### CODE REVIEW

File: src/windows/tile_inspector.c [hunk 1]
```diff
@@ -208,7 +210,43 @@ void remove_element(int index)
 	mapElement += index;
 	map_element_remove(mapElement);
 	window_tile_inspector_item_count--;
-	map_invalidate_tile_full(window_tile_inspector_tile_x, window_tile_inspector_tile_y);
+	map_invalidate_tile_full(window_tile_inspector_tile_x << 5, window_tile_inspector_tile_y << 5);
+}
+
+void rotate_element(int index)
```
Comment (IntelOrca) [inline src/windows/tile_inspector.c]: Make it static

**Comment by Broxzier** [comment]
Should I also add this in the changelog?

**Comment by IntelOrca** [comment]
@Broxzier sure, its good practise to do that.

**Comment by Broxzier** [comment]
Just rebased and resolved the conflict with english_uk.txt.

**Pull request merged**